lib/timeline.dart

```diff
@@ -5,7 +5,7 @@ import 'package:timeline_list/src/timeline_item.dart';
 import 'package:timeline_list/src/timeline_painter.dart';
 import 'package:timeline_list/timeline_model.dart';
 
-typedef IndexedTimelineModelBuilder = TimelineModel Function(
+typedef IndexedTimelineModelBuilder = TimelineModel? Function(
```

Comment:
The return type should be `TimelineModel`. As the number of models are known, the builder for models which doesn't exist shouldn't be called.